htdocs/beta/view/binet/finances.php

```diff
@@ -9,26 +9,24 @@
     </form>
   </div>
   <div class="col-lg-3">
-    <div class="bootstrap-switch bootstrap-switch-wrapper bootstrap-switch-animate bootstrap-switch-id-switch-state bootstrap-switch-off">
-      <div class="bootstrap-switch-container">
-        <?php
-          $active = "bootstrap-switch-handle-on bootstrap-switch-default";
-          $inactive = "bootstrap-switch-handle-off bootstrap-switch-primary";
-          if ($budget_view) {
-            $budget_class = $active;
-            $operation_class = $inactive;
-          } else {
-            $budget_class = $inactive;
-            $operation_class = $active;
-          }
-        ?>
-        <?php echo link_to(path("index", "budget", "", binet_prefix($binet, $term)), "Budget", $budget_class); ?>
-        <label class="bootstrap-switch-label">&nbsp;</label>
-        <?php echo link_to(path("index", "operation", "", binet_prefix($binet, $term)), "Opérations", $operation_class); ?>
-
-        <!-- C'est quoi ça ? -->
-        <input id="switch-state" type="checkbox" checked data-on-text="Budget" data-off-text="Opérations">
-      </div>
+    <div class="switch opanel">
+      <?php
+      $active = "active";
+      $inactive = "inactive";
+      if ($budget_view) {
+        $budget_class = $active;
+        $operation_class = $inactive;
+      } else {
+        $budget_class = $inactive;
+        $operation_class = $active;
+      }
+      ?>
+      <span class="left in <?php echo $budget_class ?> ">
```

Comment:
C'est quoi ce nom de class "in" ? ce n'est pas clair du tout, il faut que tu trouves quelque chose de mieux, on ne comprend pas.